docs/source/routing/self-hosted/containerization/docker.mdx

```diff
@@ -24,7 +24,9 @@ The exact image version to use depends on which release you wish to use. In the
 
 To run the router, your Docker container must have the [`APOLLO_GRAPH_REF`](/router/configuration/overview#apollo_graph_ref) and [`APOLLO_KEY`](/router/configuration/overview#apollo_key) environment variables set to your graph ref and API key, respectively.
 
-Here's a basic example of running a router image in Docker. Make sure to replace `<router-image-version>` with whichever version you want to use, such as `v1.32.0`.
+Below is a basic example of running a router image in Docker. It downloads your supergraph schema from Apollo and uses a default configuration that listens for connections on port `4000`.
+
+You can use `docker run` with the following example command:
```

Comment:
The "Below is a basic example" + new paragraph "You can use `docker run`..." threw me off a little. I was like, where is the basic example, before realizing the `docker run` example was the basic example.

```suggestion
Below is a basic example of using `docker run` to run a router image. It downloads your supergraph schema from Apollo and uses a default configuration that listens for connections on port `4000`.
```

Reply:
Will clarify that `docker run` and `docker compose` run the sample example:

```suggestion
Below is a basic example of running a router image with Docker, either with `docker run` or `docker compose`. It downloads your supergraph schema from Apollo and uses a default configuration that listens for connections on port `4000`.

You can use `docker run` with the following example command:
```